int32 sign_name
int32 x_min
int32 y_min
int32 x_max
int32 y_max
float32 distance

